open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/activedata/LayerControlPanel.java

```diff
@@ -467,58 +467,58 @@ protected void updateTileLevelControlPanel()
         final Set<TileLevelController> tlcSet = getTileLevelControllers();
         if (tlcSet != null && !tlcSet.isEmpty())
         {
-            EventQueueUtilities.runOnEDT(new Runnable()
+            EventQueueUtilities.runOnEDT(() ->
             {
-                @Override
-                public void run()
+                TileLevelController fTlc = tlcSet.iterator().next();
+                boolean hasActiveTypes = hasActiveTileDataTypes();
+                getTileHoldLevelCheckBox().setEnabled(fTlc != null && hasActiveTypes);
+                if (fTlc != null && hasActiveTypes)
                 {
-                    TileLevelController fTlc = tlcSet.iterator().next();
-                    boolean hasActiveTypes = hasActiveTileDataTypes();
-                    getTileHoldLevelCheckBox().setEnabled(fTlc != null && hasActiveTypes);
-                    if (fTlc != null && hasActiveTypes)
+                    getTileHoldLevelCheckBox().setSelected(fTlc.isDivisionOverride());
+                    int curGen = fTlc.getCurrentGeneration();
+                    if (curGen > fTlc.getMaxGeneration() && fTlc.getMaxGeneration() >= 0)
                     {
-                        getTileHoldLevelCheckBox().setSelected(fTlc.isDivisionOverride());
-                        int curGen = fTlc.getCurrentGeneration();
-                        if (curGen > fTlc.getMaxGeneration() && fTlc.getMaxGeneration() >= 0)
+                        curGen = fTlc.getMaxGeneration();
+                        for (TileLevelController c : tlcSet)
                         {
-                            curGen = fTlc.getMaxGeneration();
-                            for (TileLevelController c : tlcSet)
-                            {
-                                c.setDivisionHoldGeneration(curGen);
-                            }
+                            c.setDivisionHoldGeneration(curGen);
                         }

```

Comment:
convert to a stream:
```suggestion
                        tlcSet.forEach(c -> c.setDivisionHoldGeneration(curGen);
```